src/Microsoft.Dnx.Loader/LoadContext.cs

```diff
@@ -103,7 +113,11 @@ static LoadContext()
 
         public LoadContext()
         {
-            _contextId = Guid.NewGuid().ToString();
+        }
```

Comment:
It could be another base class but it has to be implemented on CoreCLR and on CLR and all of the implementation is the same except for the fact that we don't track the LoadContext in the static.